diagram-util/src/main/java/com/powsybl/diagram/util/forcelayout/ForceLayout.java

```diff
@@ -160,18 +161,26 @@ private void initializePoints() {
         setCenter(initialPointsCenter.orElse(new Vector(0, 0)));
 
         for (V vertex : graph.vertexSet()) {
-            Point point = new Point(
-                    center.getX() + scale * (random.nextDouble() - 0.5),
-                    center.getY() + scale * (random.nextDouble() - 0.5)
-            );
-            points.put(vertex, initialPoints.getOrDefault(vertex, point));
+            if (!fixedNodes.contains(vertex)) {
+                points.put(vertex, new Point(
```

Comment:
Here, the `initialPoints` are not used anymore. It is also weird that no test has failed after this modification (we may have not used `initialPoints` in tests)

Reply:
Indeed, I forgot the case of non-fixed initial points!